src/main/java/uk/ac/ebi/subs/agent/SamplesProcessor.java

```diff
@@ -163,4 +164,19 @@ private void announceSampleUpdate(String submissionId, List<Sample> updatedSampl
             rabbitMessagingTemplate.convertAndSend(Exchanges.SUBMISSIONS, Topics.EVENT_SAMPLES_UPDATED, updatedSamplesEnvelope);
         }
     }
+
+    //update samples with BiostudiesId as an external reference, if JWT is not present we will use super user privileges
+    protected List<Sample> updateAccessions(List<String> accessions, String biostudiesId, String jwt) {
+        String biostudiesCoreUrl = "https://www.ebi.ac.uk/biostudies/studies/";
```

Comment:
This would hardcode the prod BioStudies URL. It would be better to read it from the environment dependent application property file.